src/pynxtools/nomad/schema.py

```diff
@@ -156,7 +156,11 @@ def normalize(self, archive, logger):
             archive.workflow2 = Workflow(name=self.name)
         # steps to tasks
         act_array = archive.workflow2.tasks
-        existing_items = {(task.name, task.section) for task in act_array}
+        existing_items = {
+            (task.name, task.section)
+            for task in act_array
+            if hasattr(task, "name") and hasattr(task.section, "section")
```

Comment:
Task-s registered in archive.workflow2.tasks have both 'name' and 'section' defined. It is not a problem if they are None. In fact, we need also None-s be listed in such case.

Reply:
Initially, I got attribute errors. Later investigating the issue I see it was a bit older nomad develop branch.